<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{
	background:rgba(255,3,0,0);
	border-radius:100px;
	color: rgb(255, 255, 255);
}</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>561</width>
      <height>211</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>字体设置</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>字体</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>字体大小</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>效果</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>颜色选择</string>
      </property>
     </widget>
     <widget class="QFontComboBox" name="pic_font">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>40</y>
        <width>88</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>加粗</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>80</y>
        <width>88</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>倾斜</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>120</y>
        <width>88</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>下划线</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="pick_font_size">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>52</width>
        <height>32</height>
       </rect>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>0</y>
        <width>20</width>
        <height>171</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pick_color">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>选择颜色</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>文字位置</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>居左</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>居中</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>130</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>居右</string>
      </property>
     </widget>
     <widget class="QPushButton" name="to_left">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>向左</string>
      </property>
     </widget>
     <widget class="QPushButton" name="to_up">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>向上</string>
      </property>
     </widget>
     <widget class="QPushButton" name="to_right">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>向右</string>
      </property>
     </widget>
     <widget class="QPushButton" name="to_down">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>向下</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="pic_show">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>701</width>
      <height>391</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../PycharmProjects/python_project/spider/2021-02-17.jpg</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="preview_text">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>344</y>
      <width>516</width>
      <height>47</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>诉讼官司的嘎的</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>请在此输入您的内容</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_edit">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>311</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="file">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>导入文本文件</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="import_file">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
